pkg/docker/helper.go

```diff
@@ -19,6 +19,7 @@ import (
 
 type DockerHelper struct {
 	DockerCommand string
+	ContainerID   string
```

Comment:
could you add a comment explaining the purpose of this field please? Might not be clear in a couple of months

Reply:
Done in [499813a](https://github.com/loft-sh/devpod/pull/839/commits/499813a6d5846706879ea9f7d51b5762170a6e5f). Let me know if we want to add more details there

Reply:
great, thanks